clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/session/DistributableSession.java

```diff
@@ -87,6 +90,11 @@ public void requestDone(HttpServerExchange exchange) {
                     }
                     // If batch is closed, close session in a new batch
                     try (Batch batch = (this.batch.getState() == Batch.State.CLOSED) ? batcher.createBatch() : this.batch) {
+                        if (this.startTime != null) {
+                            // According to §7.6 of the servlet specification:
+                            // The session is considered to be accessed when a request that is part of the session is first handled by the servlet container.
+                            session.getMetaData().setLastAccessedTime(this.startTime);
```

Comment:
Is a new one of these DistributableSession objects created per request?

Must be or startTime would not change and the Batch object passed to the c'tor doesn't sound like something that would span requests.

Reply:
> Is a new one of these DistributableSession objects created per request?

Yes.